strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/security/authorization/AuthorizationConfigController.java

```diff
@@ -63,6 +64,7 @@
     private AuthorizationConfigService authorizationConfigService;
 
     @Inject
+    @XmlUserServiceQualifier
```

Comment:
Why is this a nested class?
I saw a few more qualifier classes like this. Shouldn't the be their own classes?

Reply:
Not sure I understand your question correctly but with `@XmlUserServiceQualifier` I assume that there will be `@DbUserServiceQualifier` with same `UserService` interface.